spec/models/validations/household_validations_spec.rb

```diff
@@ -363,6 +363,29 @@
       end
     end
 
+    it "does not add an error for a person aged 16-19 who is a student but not a child of the buyer" do
```

Comment:
Is it worth also having a test for "age > 19, is student, not child is fine" (with better wording), or is that covered sufficiently elsewhere?

Reply:
Sure, have added :)